### ISSUE TITLE
Checkpoint with 200 epochs?

### ISSUE THREAD

**Issue by lzyhha** [facebookresearch/simsiam#18]
Hello, can you provide the checkpoint with 200 epochs and 256 batch size in imagenet-1k? It will be better if the checkpoint include the last fc and predictor in addition to the backbone.

**Comment by endernewton** [comment]
Okay I can train one on my side.

**Comment by lzyhha** [comment]
Great, thank you.

**Comment by lzyhha** [comment]
Hello, is there any progress？

**Comment by endernewton** [comment]
Yes I have trained the model a while back but did not get chance to upload. We now have some trouble uploading the model due to the s3 bucket change. Are you able to train one on your side already?

**Comment by endernewton** [comment]
Okay got it working on my side: uploaded to https://dl.fbaipublicfiles.com/simsiam/models/200ep-256bs/pretrain/checkpoint_0199.pth.tar. 

I did a quick linear eval with the current code base and it gives 69.8 top-1 accuracy. Should be good enough to use. Closing.

**Comment by kevinbro96** [comment]
Could you share the 200 epoch training log?

**Comment by lzyhha** [comment]
这是来自QQ邮箱的自动回复邮件。This is a automatic response.
